platform/features/hyperlink/test/HyperlinkControllerSpec.js

```diff
@@ -0,0 +1,60 @@
+/*****************************************************************************
+ * Open MCT, Copyright (c) 2009-2016, United States Government
+ * as represented by the Administrator of the National Aeronautics and Space
+ * Administration. All rights reserved.
+ *
+ * Open MCT is licensed under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * http://www.apache.org/licenses/LICENSE-2.0.
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ *
+ * Open MCT includes source code licensed under additional open source
+ * licenses. See the Open Source Licenses file (LICENSES.md) included with
+ * this source code distribution or the Licensing information page available
+ * at runtime from the About dialog for additional information.
+ *****************************************************************************/
+
+define(
+    ["../src/HyperlinkController"],
+    function (HyperlinkController) {
+
+        describe("The controller for hyperlinks", function () {
+            var domainObject,
+                model,
+                controller,
+                link = "http://nasa.gov";
+            beforeEach(function () {
+                domainObject = jasmine.createSpyObj(
+                      "domainObject",
+                      ["getModel"]
+                );
+                model = jasmine.createSpyObj(
+                    "getModel",
+                    ["link","openNewTab","displayFormat"]
+                );
+                domainObject.getModel.link = link;
+                domainObject.getModel.displayFormat = "button";
+                domainObject.getModel.openNewTab = "thisTab";
+                controller = new HyperlinkController();
+            });
+            it("opens a specific given link", function () {
+                expect(domainObject.getModel.link)
+                    .toEqual("http://nasa.gov");
+            });
+            it("knows when it should open a new tab", function () {
+                expect(domainObject.getModel.openNewTab)
+                    .toEqual("thisTab");
+            });
+            it("knows when it is a button", function () {
+                expect(domainObject.getModel.displayFormat)
```

Comment:
Same as above, this should be `expect(controller.isButton()).toBe(true);`.  You'll also need a test to for "knows when it is NOT a button".